### PR TITLE
#93 New UI Improvements 

### PR DESCRIPTION
**Pull request by pvacca97**
- Fixed answers page 
- Fixed banners on other pages

### CODE REVIEW

**Comment by Lercerss** [comment]
Multiple pages (login, signup, profile, question_thread and more) are now worse off than they are currently on master. The `{% block header %}` of the templates are now surrounded by a small white border which I think should be removed. The new layout for the question_thread is good but I believe the background colors could be changed. Also, the TA's comments on the question itself have not been implemented, the question is still in its entirety located in the `{% block header %}` which can lead to confusion between it's title and content.

**Comment by alvyn279** [comment]
I like the intention of the boxes for comments and answers. We can discuss how to finalize it tomorrow. Maybe it would also be relevant to not have a boxes the complete width of the page (like for "Questions properties"), and also try to compact the questions buttons as much as possible.


Furthermore, we should not be populating the database with irrelevant about the **past**; it is important to consider modern programming technologies which are a description of the **present**. (4)
image

**Comment by alvyn279** [comment]
Please dismiss the closing of the pull request, it was an accident.

File: SA3/qa_web/templates/qa_web/login.html [hunk 1]
```diff
@@ -20,7 +19,7 @@ <h5>Incorrect username or password</h5>
     {% endif %}
 	<div align="center">
 		<h1>Welcome to QA_Web</h1>
-		<a href="/"><button>Home</button></a>
+		<a href="/"><button>Take Me Back Back Home</button></a>
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/login.html]: Fix this typo please

File: SA3/qa_web/templates/qa_web/question_thread.html [hunk 2]
```diff
@@ -13,119 +13,130 @@
 {% endblock %}
 {% block header %}
     <div class="masthead-content">
-        <br><br><br>
         <div class="container" id="question">
-            <h1 class="masthead-heading mb-0">
+            <h1  class='a-head_q' class="masthead-heading mb-0">
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/question_thread.html]: Double class

**Pull request merged**